pkg/filter/filter.go

```diff
@@ -30,16 +38,45 @@ type DispatchRule struct {
 	Rule string `toml:"rule" json:"rule"`
 }
 
+// ReplicationConfig represents config used for cyclic replication
+type ReplicationConfig struct {
+	ReplicaID       uint64   `toml:"enable" json:"enable"`
```

Comment:
```suggestion
	ReplicaID       uint64   `toml:"replica-id" json:"replica-id"`
```

Reply:
i think we should add an `enable` field into `ReplicationConfig`